src/main/java/seedu/address/model/AppointmentStatistics.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.model;
+
+/**
+ * Collects related statistics of appointment book.
+ */
+public class AppointmentStatistics {
+    private int doneToday;
+    private int upcomingToday;
+    private int doneThisWeek;
+    private int upcomingThisWeek;
+
+    /**
+     * Creates a AppointmentStatistics with the {@code totalToday}, {@code doneToday}
+     * {@code upcomingToday}, and {@code upcomingThisWeek}.
+     */
+    public AppointmentStatistics(int doneToday, int upcomingToday, int doneThisWeek, int upcomingThisWeek) {
+        this.doneToday = doneToday;
+        this.upcomingToday = upcomingToday;
+        this.doneThisWeek = doneThisWeek;
+        this.upcomingThisWeek = upcomingThisWeek;
+    }
+
+    /**
+     * Returns the number of upcoming appointments in today.
+     */
+    public int getUpcomingToday() {
+        return this.upcomingToday;
+    }
+
+    /**
+     * Returns the number of done appointments in today.
```

Comment:
Would it be better to change the `done appointments` to `appointments done`? `appointment done` might sound smoother in terms of grammar? Let me know what do you think. 👍